packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -634,6 +636,57 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
```

Comment:
A 1 or 2 liner describing what this function does would be nice. For those like me that are new to your implementation

Reply:
done